mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/dao/DaoDeleteMethodGenerator.java

```diff
@@ -105,8 +105,10 @@ public DaoDeleteMethodGenerator(
     VariableElement firstParameter = parameters.get(0);
     entityElement = EntityUtils.asEntityElement(firstParameter, typeParameters);
     hasEntityParameter = (entityElement != null);
+    final int keyParameterSize;
```

Comment:
I think we can find a better name for this. It's the number of PK columns that we want in the generated query.